jack.py

```diff
@@ -92,6 +92,38 @@
     JackTransportStarting = 3,
     JackTransportNetStarting = 4,
 } jack_transport_state_t;
+typedef uint64_t jack_unique_t;
+typedef enum {
+    JackPositionBBT = 0x10,
+    JackPositionTimecode = 0x20,
+    JackBBTFrameOffset =      0x40,
+    JackAudioVideoRatio =     0x80,
```

Comment:
I would prefer a single space around the equals sign and a comma at the end of the last line.